docs/changes/cosmology/11960.api.rst

```diff
@@ -0,0 +1,2 @@
+Non-cosmological functions are consolidated in module
```

Comment:
Unless these functions went to the top-level, I don't think this entry is necessary. I don't think we should encourage in any way to import things from lower down - always at the user's risk!

Reply:
Sounds good. None of these are publicly scoped.